tests/batch/autoscaler_test.py

```diff
@@ -15,14 +15,16 @@
 
 import mock
 import pytest
+from botocore.exceptions import EndpointConnectionError
 
 try:
     from clusterman.batch.autoscaler import AutoscalerBatch
 except ImportError:
     pytest.mark.skip('Could not import the batch; are you in a Yelp-y environment?')
+from clusterman.exceptions import PoolConnectionError
 
 
-def batch(extra_args=None, mock_sensu=True):
+def batch(extra_args=[], mock_sensu=True):
```

Comment:
default arguments should not be mutable. You should keep `extra_args=None` and do this in the body instead:
`extra_args = extra_args or []`

extra reading if you haven't heard about this yet: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments